README.md

```diff
@@ -206,6 +206,52 @@ int main(void){
 }
 ```
 
+## Run-time trace group filtering
+
+As defined the trace groups using `#define TRACE_GROUP "abcd"` in the .c/.cpp file, you can thus use this trace group for controlling the tracing.
```

Comment:
Right now this sentence is a wee bit hard to parse. Maybe something like "The trace groups you've defined using the TRACE_GROUP macro in your .c/.cpp files can be used to control tracing at run-time."?

Reply:
In my mind it was something like that, the end result in the file is incomprehensible gibberish. Right you are.